odoo/addons/base/tests/test_res_users.py

```diff
@@ -157,6 +157,18 @@ def test_change_user_company(self):
             "the company_id of the partner_id shall be updated"
         )
 
+    def test_user_phone(self):
```

Comment:
To make it a bit more generic ?
```suggestion
    def test_user_self_update(self):
```